FoodFinder/ViewModels/MapViewModel.swift

```diff
@@ -10,78 +10,21 @@ import Foundation
 import MapKit
 
 protocol MapViewModelDelegate: class {
-    func shouldSetRegion(_ region: MKCoordinateRegion)
-    func shouldAddAnnotations(_ annotations: [MKPointAnnotation])
-//    func didUpdateLocations(_ location: [CLLocation])
     func didFetchRestaurants(_ restaurants: [Restaurant])
 }
```

Comment:
I'm able to simplify this whole protocol because a lot of it isn't really necessary once this is all moved around.